app/controllers/ConfirmationController.scala

```diff
@@ -43,7 +43,7 @@ class ConfirmationController @Inject()(
                                         getData: DataRetrievalAction,
                                         requireData: DataRequiredAction,
                                         val controllerComponents: MessagesControllerComponents,
-                                        val paperlessPreferenceConnector: PaperlessPreferenceConnector,
+                                        val paperlessPreferenceConnector: PaperlessPreferenceConnectorImpl,
```

Comment:
This needs to be `PaperlessPreferenceConnector` not the implementation class

Reply:
Done